.changeset/serious-fishes-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-github-pull-requests': major
```

Comment:
I think we might get away with minor here. There doesn't seem to be breaking changes.
Could be good to bump major for vanity reasons though...

Reply:
I agree, I wanted it to be a minor change, but missed it :D